src/components/PublishContainer/PublishPage/PublishPage.tsx

```diff
@@ -29,21 +29,16 @@ export const PublishPage: FunctionComponent<PublishPage> = ({ config }) => {
   const documentsPublished = publishState === "CONFIRMED";
 
   const createAnotherDoc = (): void => {
-    const nextForms = [...forms];
-    nextForms.splice(nextForms.length - 1, 1);
-    setForms(nextForms);
+    setForms([]);
     setActiveFormIndex(undefined);
   };
 
   const iAmDone = (): void => {
```

Comment:
Didnt notice this naming earlier, think its weird that it's named this way. Would suggest either onDone or something similar.

Reply:
Also, wonder if its clearer to the user if we use "Logout" as the button label instead? As a user im not sure what im expecting when i click on this.

Reply:
ok, true will change it to "logout".